### ISSUE TITLE
generate steps

### ISSUE THREAD

**Issue by valmadrid** [learn-co-students/applying-gradient-descent-lab-data-science-intro-000#4]
arrangement of args is not correct when function was called for iterations.  this generated a different m, b and rss

def generate_steps(**m, b**, number_of_steps, movies, learning_rate)

iterations = generate_steps(**initial_regression_line['b'], initial_regression_line['m'],** 100, scaled_movies, .0001) or [{'m': 'uncomment generate_steps method', 'b': 'uncomment generate_steps method '}]
